Car_owners/tests/test_models.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from application.models import Owner, Car
+
+
+@pytest.fixture
+def model_data() -> dict[str, Owner | Car]:
+    owner = Owner.objects.create(name="Adam", surname="Knafel", phone="123456789")
+    car = Car.objects.create(
```

Comment:
do these objects stay in the database when running tests?

Reply:
Only for one particular test. Each test will run in its own database setup that will be rolled back at the end of the test.

Reply:
where is the rollback? Better to use smth like this https://pytest-django.readthedocs.io/en/latest/database.html